packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md

```diff
@@ -0,0 +1,75 @@
+
+This screen contains settings for all information that flows from your application upstream into Okta.
+
+Click **Edit** to make changes to the following sections.
+
+* **General** &mdash; Use this section to schedule imports and dictate a username format for imported users. You can also define a percentage of acceptable assignments before the [import safeguards](https://help.okta.com/en/prod/okta_help_CSH.htm#csh-eu-import-safeguard) feature is automatically triggered.
+
+    If the Okta username is overridden due to mapping from a provisioning-enabled app, the custom mapping appears in this section.
+
+* **User Creation & Matching** &mdash; Matching rules are used when importing users from all applications and directories that allow importing. Establishing matching criteria allows you to specify how an imported user should be defined as a new user or mapped to an existing Okta user.
+
+  * **Imported user is an exact match to Okta user if** &mdash; The matching criteria that establishes whether an imported user exactly matches an existing Okta user. Choose any combination from the list of options to establish your criteria. For the new imported user to be considered an exact match, each option that you select must be true. Note that if you choose the third option, the first and second choices are disabled.
+
+  * **Allow partial matches** &mdash; Partial matching occurs when the first and last name of an imported user match those of an existing Okta user, but the username or email address do not.
+
+  * **Confirm matched users** &mdash; Select to automate the confirmation or activation of existing users. If the option isn't selected, matches are confirmed manually.
+
+  * **Confirm new users** &mdash; Select to automate the confirmation or activation of a newly imported user. If this option is selected, you can clear it during import confirmation. Note that this feature doesn't apply for users who already exist in Okta.
+
+* **Profile & Lifecycle Mastering** &mdash; Use this section to allow the current application to profile master Okta users. If enabled, the application appears in the list of profile masters on the Profile mastering page.
+
+  * **Allow `<app>` to master Okta users** &mdash; Enable mastery and determine what happens when a user is deactivated or reactivated in an application. Only the highest priority profile master for that Okta user can deactivate or suspend an Okta user. To verify the highest priority profile master, review the Profile mastering page.
+
+  * **When a user is deactivated in the app** &mdash; Select **Do Nothing** to prevent activity in the application from controlling the user life cycle. This still allows profile master control of attributes and mappings. Otherwise, you can choose between deactivating or suspending the user.
+
+  * **When a user is reactivated in the app** &mdash; Decide if suspended or deactivated Okta users should be reactivated when they have been reactivated in the application.
```

Comment:
I don't understand this sentence: So, a suspended or deactivated Okta user can be reactivated and **then** the admin has to decide if they should be reactivated?

Reply:
Looking at this text, it's not clear what is being reactivated where. I have changed it to:
"Determines if deactivated or suspended Okta user profiles that are assigned to your Okta integration should also be reactivated when they have been reactivated in your downstream application."